.evergreen/scripts/abi-compliance-check-test.sh

```diff
@@ -117,18 +118,29 @@ fi
 # Allow task to upload the HTML report despite failed status.
 echo "Generating stable ABI report..."
 pushd cxx-abi
-if ! abi-compliance-checker -lib mongo-cxx-driver -old old.xml -new new.xml 2>&1; then
-  : # CXX-2812: enable code below once stable ABI symbols exist in the base commit libraries.
-  # declare status
-  # status='{"status":"failed", "type":"test", "should_continue":true, "desc":"abi-compliance-checker emitted one or more errors"}'
-  # curl -sS -d "${status:?}" -H "Content-Type: application/json" -X POST localhost:2285/task_status || true
+declare ret
+abi-compliance-checker -lib mongo-cxx-driver -old old.xml -new new.xml 2>&1 && ret="$?" || ret="$?"
+if [[ "${ret:?}" -gt 0 ]]; then
```

Comment:
Swap if blocks. I expect the `-gt 1` block will never be entered (since the `-gt 0` will be true too).